src/imitation/scripts/data_collect.py

```diff
@@ -46,16 +48,19 @@ def data_collect(_seed: int,
       parallel: If True, then use DummyVecEnv. Otherwise use SubprocVecEnv.
       normalize: If True, then rescale observations and reward.
       make_blank_policy_kwargs: Kwargs for `make_blank_policy`.
-
       rollout_save_interval: The number of training updates in between
           intermediate rollout saves. If the argument is nonpositive, then
           don't save intermediate updates.
       rollout_save_final: If True, then save rollouts right after training is
           finished.
-      rollout_save_n_samples: The minimum number of timesteps saved in every
-          file. Could be more than `rollout_save_n_samples` because trajectories
-          are saved by episode rather than by transition.
-
+      rollout_save_n_timesteps: The minimum number of timesteps saved in every
+          file. Could be more than `rollout_save_n_timesteps` because
+          trajectories are saved by episode rather than by transition.
+          Must set exactly one of `rollout_save_n_timesteps`
+          and `rollout_save_n_episodes`.
+      rollout_save_n_episodes: The number of episodes saved in every
```

Comment:
Can we add some input validation for this so it fails fast? Right now it will only fail when we actually try to save the rollouts, which in the worst case could be at the very end of policy training.

Reply:
Good call